app/src/app/components/stix/stixid-property/stixid-property.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="stix-header text-deemphasis">
+    <div class="content" [ngClass]="{'icon-button': config.object['stixID']}">
+        {{config.object["stixID"]}}
+        <button mat-icon-button class="button-select" *ngIf="config.object['stixID']" [cdkCopyToClipboard]="stixIdLink"
```

Comment:
Since the STIX ID is being displayed in the adjacent div, we should be able to safely assume that it exists at this point in the component. I'd recommend moving the *ngIf condition to the AppSubheaderComponent to ensure that the entire app-stixid-property component is only rendered when there is a STIX ID present.